README.md

```diff
@@ -51,6 +51,7 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 Name     | Description | OS
 ---------|-------------|----
+bcache | Exposes bcache statistics | Linux
```

Comment:
We prefer to turn reasonable collectors on by default.  I think it would be fine to enable this.